config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
- # root "items#index"
+ root to: "vendors#index"
+
+ resources :vendors, only: [:index, :show]
+
```

Comment:
eventually I think we will want items nested, this should be fine for now though.